tests/D2L.CodeStyle.Analyzers.Test/Specs/ImmutableGenericDeclarationAnalyzer.cs

```diff
@@ -238,36 +238,40 @@ class GenericMethodClass {
 			}
 		}
 
-		public class ImmutableGenericEnum {

```

Comment:
The public modifier wasn't necessary and was inconsistent with the rest of the file, so I removed it while I was here.